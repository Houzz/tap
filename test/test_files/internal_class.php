<?php

$x = new DateTime();
$timestamp = $x->getTimestamp();
__tap_check($timestamp, "int");

$y = new DateTime2();   // class not found

