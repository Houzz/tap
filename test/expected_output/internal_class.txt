7 E_CLASS_NOT_FOUND
